oppia/oppia
# Fix #4065: New  editor logout handler

- created new logout handler ''EditorLogoutHandler" in editor.py.

- added route in  "main.py".

- modified "create_logout_url()" in "gae_current_user_services.py" to check for exp_editor page and process accordingly.

Testing:
- login and click on create. now logout: should redirect to /library.

- login and click on create. publish the exploration. now logout: should redirect to same page.
